llvm/lib/Support/Caching.cpp

```diff
@@ -145,7 +145,8 @@ Expected<FileCache> llvm::localCache(const Twine &CacheNameRef,
       // ensures the filesystem isn't mutated until the cache is.
       if (std::error_code EC = sys::fs::create_directories(
               CacheDirectoryPath, /*IgnoreExisting=*/true))
-        return errorCodeToError(EC);
+        return createStringError(EC, Twine("Can't create cache directory ") +
```

Comment:
https://llvm.org/docs/CodingStandards.html#error-and-warning-messages The majority of places prefer no capitalization.